content/_layouts/post.html.haml

```diff
@@ -27,15 +27,19 @@ section: blog
       = page.date.strftime('%B %-d, %Y')
 
     .share-container
-      %button.share-toggle-button.twitter-share-button
-        %img.post-icon{ src: "/images/symbols/share-button-post.png", alt: "Post" }
+      %button.app-button.app-button--primary
+        = File.read("content/images/symbols/share-button-post.svg")
         Share
       .share-buttons
-        %a.twitter-share-icon{:href => "https://twitter.com/intent/tweet?text=#{CGI::escape(page.title)}&url=#{CGI::escape("https://www.jenkins.io" + expand_link(page.url))}", 'rel' => 'noreferrer nofollow', 'target' => '_blank'}
-        %a.linkedin-share-icon{:href => "https://www.linkedin.com/sharing/share-offsite/?url=#{CGI::escape("https://www.jenkins.io" + expand_link(page.url))}", 'rel' => 'noreferrer nofollow', 'target' => '_blank'}
-        %a.mastodon-share-icon{:href => "https://mastodon.social/share?text=#{CGI::escape(page.title + " " + "https://www.jenkins.io" + expand_link(page.url))}", 'rel' => 'noreferrer nofollow', 'target' => '_blank'}
-        %a.bluesky-share-icon{:href => "https://bsky.app/intent/compose?text=#{CGI::escape(page.title + " " + "https://www.jenkins.io" + expand_link(page.url))}", 'rel' => 'noreferrer nofollow', 'target' => '_blank'}
-        
+        %a{:href => "https://twitter.com/intent/tweet?text=#{CGI::escape(page.title)}&url=#{CGI::escape("https://www.jenkins.io" + expand_link(page.url))}", :class => "app-button app-button--square app-social-media-button-x", :'data-tooltip' => "𝕏 (formerly Twitter)", 'rel' => 'noreferrer nofollow', 'target' => '_blank'}
+          = File.read("content/images/symbols/x.svg")
+        %a{:href => "https://www.linkedin.com/sharing/share-offsite/?url=#{CGI::escape("https://www.jenkins.io" + expand_link(page.url))}", :class => "app-button app-button--square app-social-media-button-linkedin", :'data-tooltip' => "LinkedIn", 'rel' => 'noreferrer nofollow', 'target' => '_blank'}
```

Comment:
I think the tooltips should have the whole action there, e.g. `Share on $social-media`
```suggestion
        %a{:href => "https://www.linkedin.com/sharing/share-offsite/?url=#{CGI::escape("https://www.jenkins.io" + expand_link(page.url))}", :class => "app-button app-button--square app-social-media-button-linkedin", :'data-tooltip' => "Share on LinkedIn", 'rel' => 'noreferrer nofollow', 'target' => '_blank'}
```